AirBench/Models/User.cs

```diff
@@ -8,22 +8,25 @@ namespace AirBench.Models
 {
     public class User
     {
-        public User() { }
+        public User() {
+            Benches = new List<Bench>();
+            Reviews = new List<Review>();
+        }
 
         public User(int id, string userName, string hashedPassword)
         {
             Id = id;
             UserName = userName;
             HashedPassword = hashedPassword;
         }
-
+  
         public int Id { get; set; }
         [Required, MaxLength(255)]
         public string UserName { get; set; }
         [Required, MaxLength(255)]
         public string HashedPassword { get; set; }
 
         public List<Bench> Benches { get; set; }
-        public List<Review> Reviews { get; set; }
+        public virtual List<Review> Reviews { get; set; }
```

Comment:
Be careful with using `virtual` collection properties on your entity models. This enables something called "lazy loading", which often results in poor performance. It's preferable to not use any `virtual` properties and eagerly load related entities using the `Include` operator.